package.json

```diff
@@ -122,5 +122,9 @@
   },
   "release": {
     "preset": "conventionalcommits"
+  },
+  "dependencies": {
+    "@linusborg/vue-simple-portal": "^0.1.5",
+    "@popperjs/core": "^2.10.2"
```

Comment:
These should be peer dependencies so they're not duplicated in the consuming app.